### PR TITLE
fix: pip chat option ui, audio mode ui

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

**Review by eswarclynn — APPROVED** [review]

File: packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const PIPChatOption = ({ openChat, showPIPChat }: { openChat: () => void;
   }
   return (
     <Dropdown.Item onClick={openChat} data-testid="pip_chat_btn">
-      <ExternalLinkIcon height={18} width={18} style={{ padding: '0 $2' }} />
+      <ExternalLinkIcon height={18} width={18} style={{ padding: '0 3px', boxSizing: 'content-box' }} />
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx:12]: why is this needed? doesn't seem to be there for other icons.

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx:12]: also no padding is there for other icons.

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx:12]: The external link icon does not have padding. Will add it again in this PR with the spacing

File: packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const PIPChatOption = ({ openChat, showPIPChat }: { openChat: () => void;
   }
   return (
     <Dropdown.Item onClick={openChat} data-testid="pip_chat_btn">
-      <ExternalLinkIcon height={18} width={18} style={{ padding: '0 $2' }} />
+      <ExternalLinkIcon height={24} width={24} />
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx:12]: default is 24 does it have to be passed?

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/PIP/PIPChatOption.tsx:12]: Not needed, removing. I thought 20 was the default

**Review by raviteja83 — APPROVED** [review]
